src/coffee/repeater-task-queue.coffee

```diff
@@ -0,0 +1,73 @@
+{ Repeater } = require 'sphere-node-utils'
+Promise = require 'bluebird'
+util = require 'util'
+TaskQueue = require './task-queue'
+
+# Response messages which will be handled by `RepeaterTaskQueue`
+retryKeywords = [
```

Comment:
Hmm, this looks very fragile. Usually you retry based on the status code. Having hard coded pieces of strings is really not flexible if e.g.: the message changes.
Can we get rid of this list?